src/sharding/shard.ts

```diff
@@ -1,12 +1,14 @@
 import { ShardingManager } from 'discord.js';
 import { config } from 'dotenv';
+import { createLogger } from 'winston';
 // @ts-ignore When running GitHub Actions, the config file isnt available
 import Configuration from '../../config.json';
-import { Logger } from '../bot/utils/Logger';
+import { Initializer } from '../bot/utils/Initializer';
 
 config({ path: '../.env' });
 
-const logger = new Logger();
+const logger = createLogger({ handleRejections: true, handleExceptions: true });
+Initializer.loadWinstonLogger(logger);
```

Comment:
```suggestion
Initializer.loadWinstonLogger(logger, 0);
```

Reply:
Wait shard 0? Shouldn't be the shard manager?